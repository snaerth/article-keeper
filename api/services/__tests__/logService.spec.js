import logError from '../logService';

describe('Test for log service', () => {
  // FileExists
  test('Log error to mongodb', async () => {
    try {
      const res = await logError(new Error('Ãžetta er villa'));
      expect(res).toBe(undefined);
    } catch (error) {
      expect(error).toThrowErrorMatchingSnapshot();
      throw new Error(error);
    }
  });
});
