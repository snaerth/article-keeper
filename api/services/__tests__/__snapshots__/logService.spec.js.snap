// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for log service Log error to mongodb 1`] = `"received is not a function"`;
